conf/defaults.yaml

```diff
@@ -140,7 +140,8 @@ supervisor.memory.capacity.mb: 3072.0
 supervisor.cpu.capacity: 400.0
 
 ### worker.* configs are for task workers
-worker.childopts: "-Xmx768m"
+worker.heap.memory.mb: 768
+worker.childopts: "-Xmx%HEAP-MEM%m"
```

Comment:
The reason for this is backwards compatibility + flexibility.  If an admin has set their own worker.childopts for a given supervisor to say 2GB, we want that to have higher priority over what we calculate.  Similarly if someone sets topology.worker.childopts it needs to have higher priority over the other two. But what if the topology wants to test out the resource aware scheduler on just their topology?  With this they can set topology.worker.childopts to "-Xmx%HEAP-MEM%m" and override the hard coded 2GB that was there before.